internal/game/game.go

```diff
@@ -0,0 +1,45 @@
+package game
+
+import (
+	"github.com/gorilla/websocket"
+	"github.com/oklog/ulid/v2"
+	"go.uber.org/zap"
+)
+
+type Game struct {
+	rooms []*room
```

Comment:
This seems... odd? Why are the rooms part of the Game object? Shouldn't a room contain a game object?

Reply:
Also: The room only exist in RAM, right? Probably alright for now, but maybe we should keep in mind that we might want to have more than one instance of this server at some point.

Reply:
That's something I definitely want to do, but it's not important at this point. First of all, the game should work, then we can think about something like that

Reply:
I understand what you mean. I'm not happy with the name either. How about something like `GameManager`?

Reply:
Or RoomManager?

Reply:
Hmm... I think either is fine for now. We can still rename it later on anyway.

Reply:
fixed